index.html

```diff
@@ -0,0 +1,215 @@
+            Symbols are often used to add unique property keys to an object that won't collide with keys any other code might add to the object, and which are hidden from any mechanisms other code will typically use to access the object.
+             That enables a form of weak <a href="https://developer.mozilla.org/en-US/docs/Glossary/Encapsulation">encapsulation</a>, or a weak form of <a href="https://en.wikipedia.org/wiki/Information_hiding">information hiding</a>.</p>
+          <p>Every <code>Symbol()</code> call is guaranteed to return a unique Symbol. Every <code>Symbol.for("key")</code> call will always return the same Symbol for a given value of <code>"key"</code>. When <code>Symbol.for("key")</code> is called, if a Symbol with the given key can be found in the global Symbol registry, that Symbol is returned. Otherwise, a new Symbol is created, added to the global Symbol registry under the given key, and returned.</p>
+        </ul>
+      </section>
+
+
+      <section class="main-section" id="Text_processing">
+        <header class="subtitle">
+          <h2>Text processing</h2>
+        </header>
+        <ul>
+          <li><a href="#Text_processing">String</a></li>
+          <p>The <code>String</code> object is used to represent and manipulate a sequence of characters. </p>
+          <h4>Creating strings:</h4>
+          <p>Strings can be created as primitives, from string literals, or as objects, using the <a href="https://devdocs.io/javascript/global_objects/string/string">String()</a> constructor: </p>
+          <pre class="line-numbers">
+          <code>
+            const string1 = "A string primitive";
+            const string2 = 'Also a string primitive';
+            const string3 = `Yet another string primitive`;
+            const string4 = new String("A String object");
+            </code>
+            </pre>
+          <li><a href="#Text_processing">RegExp</a></li>
+          <p>The <code>RegExp</code> object is used for matching text with a pattern.</p>
+          <p>There are two ways to create a RegExp object: a literal notation and a constructor.</p>
+<ul>    <li>The literal notation's parameters are enclosed between slashes and do not use quotation marks.</li>
+<li>The constructor function's parameters are not enclosed between slashes but do use quotation marks.</li></ul>
+<p>The following three expressions create the same regular expression object:</p>
+<pre class="line-numbers">
+  <code>
+let re = /ab+c/i; // literal notation
+let re = new RegExp('ab+c', 'i') // constructor with string pattern as first argument
+let re = new RegExp(/ab+c/, 'i') // constructor with regular expression literal as first argument (Starting with ECMAScript 6)
+</code></pre>
+        </ul>
+      </section>
+
+    </main>
+<div id="footer">

```

Comment:
A better practice than setting a footer div with the id="footer", is to use the `<footer>` tag directly for the footer section of the html